config/initializers/webauthn.rb

```diff
@@ -0,0 +1,24 @@
+WebAuthn.configure do |config|
+  # This value needs to match `window.location.origin` evaluated by
+  # the User Agent during registration and authentication ceremonies.
+  config.origin = ENV["WEBAUTHN_ORIGIN"] || "http://localhost:3000"
```

Comment:
New `WEBAUTHN_ORIGIN` environment variable? Shouldn't this equal the "local (or web) domain" setting set elsewhere?

Reply:
I tried this with `LOCAL_DOMAIN` but tests are failing – which make sense if you take a look at the environment variables for tests: `LOCAL_DOMAIN` should be `http://test.host`

Reply:
Admins aren't going to set a new environment variable just for this which must be identical with an existing one anyway.

Try using `Rails.config.x.web_domain` for this

Reply:
>Admins aren't going to set a new environment variable just for this which must be identical with an existing one anyway.
>
>Try using Rails.config.x.web_domain for this, though you'll also need to use Rails.config.x.use_https since the other variable does not contain the scheme part.

Got it! Then I will try to do it in that way 🙂

Reply:
>Try using Rails.config.x.web_domain for this, though you'll also need to use Rails.config.x.use_https since the other variable does not contain the scheme part.

Great! I got it to work using `Rails.configuration.x.web_domain` and `Rails.configuration.x.use_https`.

>I tried this with LOCAL_DOMAIN but tests are failing – which make sense if you take a look at the environment variables for tests: LOCAL_DOMAIN should be http://test.host

Don't pay attention to this, it works properly using the environment variables `LOCAL_DOMAIN` and `LOCAL_HTTPS`.
I just forgot to make changes in the tests that I was manually setting the domain of the `FakeClient` to be `http://test.host`.

See: https://github.com/tootsuite/mastodon/pull/14466/commits/830000bcb96cc1ced8643dca7a8fa85eb4ffc2e6